packages/core/src/storages/request_queue_v2.ts

```diff
@@ -111,6 +112,10 @@ export class RequestQueue extends RequestProvider {
     override async fetchNextRequest<T extends Dictionary = Dictionary>(): Promise<Request<T> | null> {
         checkStorageAccess();
 
+        if (this.queuePausedForMigration) {
```

Comment:
@drobnikj it may not be clear, but this is also set to true if we receive an `ABORTING` event